release.config.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  branches: ['master'],
+  plugins: [
```

Comment:
I'd recommend making these changes:
1. `yarn add -D conventional-changelog-conventionalcommits`
2. add this entry to the beginning of this plugin list:
    `['@semantic-release/commit-analyzer', { preset: 'conventionalcommits' }],`
3. Create a new file at `.github/lifeomic-probot.yml` that looks like this:
    ```yml
    enforceSemanticCommits: true
    ```

Why the probot thing: the probot config will perform nice checks on PRs to ensure every PR contains a semantic commit, and flag major version bumps.

Why "conventional-commits":
- every other project in our org that uses `semantic-release` uses this commit spec, AFAIK "conventional commits" spec, and
- the probot config currently only supports that spec